core/src/main/kotlin/entity/Guild.kt

```diff
@@ -354,12 +363,15 @@ public class Guild(
     /**
      * Gets the banner url in the specified format.
      */
+    @Deprecated("Old method", ReplaceWith("Guild#banner"), DeprecationLevel.WARNING)
```

Comment:
```suggestion
    @Deprecated("Old method", ReplaceWith("this.banner"), DeprecationLevel.WARNING)
```
this way an automatic replacement can be performed by intellij (applies to other `ReplaceWith`s as well)